tests/virt_autotest/login_console.pm

```diff
@@ -198,8 +198,8 @@ sub login_to_console {
             save_screenshot;
             #start upgrade
             if (check_var('VIDEOMODE', 'text')) {
-                if (lc(get_var('VERSION_TO_INSTALL', '')) eq '12-sp5' and lc(get_var('UPGRADE_PRODUCT', '')) eq 'sles-15-sp6') {
-                    #DIAPLAY= might be culprit that prevents host upgrade from proceeding at SCC registration. Please refer to bsc#1218798.
+                if ((lc(get_var('VERSION_TO_INSTALL', '')) eq '12-sp5' and lc(get_var('UPGRADE_PRODUCT', '')) eq 'sles-15-sp6') or (lc(get_var('VERSION_TO_INSTALL', '')) eq '12-sp5' and lc(get_var('UPGRADE_PRODUCT', '')) eq 'sles-15-sp7')) {
```

Comment:
It includes prj2 on x86 too, thanks.  LGTM but the if condition can become shorter with `=~ /xx|xx/`.

Reply:
thanks, updated